pkg/client/mongodb/mongodb.go

```diff
@@ -0,0 +1,48 @@
+package mongodb
+
+import (
+	"context"
+	"github.com/joho/godotenv"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+	"os"
+	"time"
+)
+
+//var DB *mongo.Client = ConnectDB()
+
+var DB, _ = ConnectDB()
```

Comment:
Can you avoid global variables? Here, you are also ignoring an error.
Some malicious package for example can switch mongodb.DB=nil causing panics that are hard to track. If you want to really have a singleton, just add a function DB() that will return the db connection. Also, before you start the web server, preferably you would like to make a connection and test that you can actually ping the db. This way, we have no idea if connection is ok

Reply:
If you really want to make it like this, and not call it from the main, you should leverage `init()` function